pymc_marketing/clv/models/basic.py

```diff
@@ -78,19 +97,125 @@ def _fit_MAP(self, **kwargs):
         map_strace.record(map_res)
         map_strace.close()
         trace = MultiTrace([map_strace])
-        return pm.to_inference_data(trace, model=model)
+        idata = pm.to_inference_data(trace, model=model)
+        self.set_idata_attrs(idata)
+        return idata
```

Comment:
Why doesn't this save to self.idata and mcmc does?

Reply:
fixed, now they both act the same.